guide/getting-started/visual-language.md

```diff
@@ -188,7 +189,7 @@ Magazines, blogs, and other publications often hire professional illustrators to
 
 ## The expression scale
 
-A good way to think about the visual language of Bitcoin is of a scale that ranges from minimal to highly expressive. Despite the images below looking and communicating so differently, they can all still be recognized as capturing Bitcoin. One of the great aspects of public, community-owned identities like the one Bitcoin has, is that all of these expressions are equally legitimate and valid. This provides enormous creative freedom to designers that we should explore and take advantage of.
+Think about Bitcoin’s visual language as a scale that ranges from muted to highly expressive. These images look, but they all capture bitcoin in some way. Public, community-owned identities like Bitcoin make them all equally valid and canonical. This provides enormous creative freedom to designers and illustrators with their own vision for Bitcoin.
```

Comment:
```suggestion
Think about Bitcoin’s visual language as a scale that ranges from muted to highly expressive. These images look very different, but they all capture Bitcoin in some way. As the identity of Bitcoin is public and community-owned, all these representations are equally valid. This provides enormous creative freedom to designers and illustrators with their own visions.
```
Fixed broken sentence.

> Public, community-owned identities like Bitcoin make them all equally valid and canonical.

Struggled with this sentence, so I tried to come up with a simpler variation.

Remove the last Bitcoin, as it was the fourth appearance of the word in this short paragraph.